health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/EditRegister.js

```
@@ -15,306 +15,283 @@ import EditAttendanceManagementTable from "../../components/EditAttendanceManage
+                variation="link"
+                style={{ whiteSpace: "nowrap", width: "auto" }}
+              />
+            </span>
+          </div>
+        </Card>
+
+        {/* ---- Attendance Edit Table ---- */}
+        <Card className="bottom-gap-card-payment">
+          <div className="card-heading">
+            <h2 className="card-heading-title"></h2>
+            <Button
```

Biome (2.1.2)

[error] Provide screen reader accessible content when using heading  elements.

All headings on a page should have content that is accessible to screen readers.

(lint/a11y/useHeadingContent)